artifactory/commands/helm/flags.go

```diff
@@ -0,0 +1,251 @@
+package helm
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+// helmFlags contains all parsed flags and arguments from os.Args
+type helmFlags struct {
+	commandName string
```

Comment:
it is more clear if it is named as helmSubCommand
```suggestion
	helmSubCommand string
```